### PR TITLE
US107060 Added two new entities

### PR DESCRIPTION
**Pull request by ciss1995**

### CODE REVIEW

**Comment by m6jones** [comment]
Approved after lint fixes

**Comment by ctwomey1** [comment]
Would be swell to get some tests added :)

**Comment by ciss1995** [comment]
@ctwomey1 Ok to add tests in another PR?

**Comment by ctwomey1** [comment]
I think test coverage could save us in the long run, but w/e you feel is appropriate

**Comment by m6jones** [comment]
Adding tests isn't too bad. Sample tests for the enrollment entity.

**Comment by ciss1995** [comment]
Tests should stop failing for IE11 After this https://github.com/Brightspace/node-siren-parser/pull/26

File: src/userSettings/UserSettingsEntity.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import { Entity } from '../es6/Entity.js';
+import { Rels } from '../hypermedia-constants';
+import { Actions } from 'd2l-hypermedia-constants';
```
Comment (svanherk) [inline src/userSettings/UserSettingsEntity.js]: Can use the Actions from the local `hypermedia-constants` file

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "lie": "^3.3.0",
     "polymer-cli": "^1.9.1",
     "wct-browser-legacy": "^1.0.1",
-    "whatwg-fetch": "^2.0.0"
+    "whatwg-fetch": "^2.0.0",
+    "d2l-hypermedia-constants": "^6"
```
Comment (svanherk) [inline package.json]: Can remove this if we use the local file

**Pull request merged**